src/view/screens/Profile.tsx

```diff
@@ -522,3 +524,77 @@ const styles = StyleSheet.create({
     textAlign: 'center',
   },
 })
+
+const shouldExposeToGate2 = Math.random() < 0.2
+
+// --- Temporary: we're testing our Statsig setup ---
+let TestGates = React.memo(function TestGates() {
+  const gate = useGate()
+
+  useEffect(() => {
+    logEvent('test:all:always', {})
+    if (Math.random() < 0.2) {
+      logEvent('test:all:sometimes', {})
+    }
+    if (Math.random() < 0.1) {
+      logEvent('test:all:boosted_by_gate1', {
+        reason: 'base',
+      })
+    }
+    if (Math.random() < 0.1) {
+      logEvent('test:all:boosted_by_gate2', {
+        reason: 'base',
+      })
+    }
+    if (Math.random() < 0.1) {
+      logEvent('test:all:boosted_by_both', {
+        reason: 'base',
+      })
+    }
+  }, [])
+
+  return [
+    gate('test_gate_1') ? <TestGate1 /> : null,
+    shouldExposeToGate2 && gate('test_gate_2') ? <TestGate2 /> : null,
```

Comment:
Expose a smaller % to this gate (note separate sessions increase the probability of exposure because `Math.random` for `shouldExposeToGate2` is at module level)